### DISCUSSION TITLE
Please don't become bloatware like postman

### DISCUSSION THREAD

**Discussion by stepacool** [Kong/insomnia#6680] [Feedback]
The latest update of 8.x version feels like postman, bloatware, agressive login page, poorly working curl import, so much garbage compared to the previous version. I understand you want monetization but please understand that your product is becoming worse and the very thing why people turn away from postman.

**Comment by subnetmarco** [comment]
Thanks, we will try to strike the right balance. We reintroduced local projects with 8.3 in addition to Cloud and Git, so this is already a differentiator (https://github.com/Kong/insomnia/discussions/6626)
